### DISCUSSION TITLE
Questions related to Healthstack project

### DISCUSSION THREAD

**Discussion by Jawwad-Fida** [Jawwad-Fida/HealthStack-System#483] [General]
Hi all!

If you have any questions or suggestion related to our Healthstack project, do let us know here.

Thank you!

**Comment by KathpaliaChirag** [comment]
hey there, looked at your work that's really amazing... I'm working on something similar, A college ERP project could you tell me how did you guys configured database .... I'm using PostgreSQL... but not sure how to implement it for a live demonstration.... as long as data is in my pc i can surely watch it and work on it... but I'm confused how to link the database to GitHub so my other 2 friends can as well make changes in it at ease...

**Reply by Jawwad-Fida** [reply]
I used ngrok server to share my localhost server with my teammates. So basically, I shared the database on my local pc with them. When they made any changes to the database, those changes also came to mine. 

U can refer to the link on [ngrok ](https://www.sitepoint.com/use-ngrok-test-local-site/)
